### PR TITLE
feature/bS-3759-Align readme

### PR DESCRIPTION
**Pull request by Ofir-Lapid**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,28 @@
...
+<p align="center"> <img src="Data%20flow.png" alt="drawing" width="60%"/></p>
 
 ## Usage
+
+Before you start we recommend completing the onboarding tutorial.
+
+---
+**Important note**
+
+To simplify the setup process, the following example uses a local [build.security pdp instance](https://docs.build.security/policy-decision-points-pdp/pdp-deployments/standalone-docker-1).
```
Comment (yashtewari) [inline README.md:19]: - pdp -> PDP

File: README.md [hunk 2]
```diff
@@ -1,13 +1,28 @@
...
+If you are already familiar with how to run your PDP (Policy Decision Point), You can also run a pdp on you environment (Dev/Prod, etc).
```
Comment (yashtewari) [inline README.md:20]: - pdp -> PDP

File: README.md [hunk 3]
```diff
@@ -1,13 +1,28 @@
 
 # opa-express-middleware
+<p align="center"><img src="Logo-build.png" class="center" alt="build-logo" width="30%"/></p>
+
 ## Abstract
-Node.js express middleware to authorize API requests using a 3rd party policy engine (OPA) as the Policy Decision Point (PDP).
-If you're not familiar with OPA, please [learn more](https://www.openpolicyagent.org/).
+[build.security](https://docs.build.security/) provides simple development and management of the organization's authorization policy.
```
Comment (yashtewari) [inline README.md:6]: - of the organization's -> for your organization's

File: README.md [hunk 4]
```diff
@@ -1,13 +1,28 @@
...
+opa-express-middleware is a Node.js express middleware intended for performing authorizing requests against build.security pdp/[OPA](https://www.openpolicyagent.org/).
```
Comment (yashtewari) [inline README.md:7]: - express middleware -> Express framework middleware
- intended for performing authorizing requests -> intended to perform authz
- pdp -> PDP

**Review by yashtewari — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (Ofir-Lapid) [reply README.md:7]: Changed to - intended for performing autorization requests...

**Review by yashtewari — APPROVED** [review]

**Pull request merged**